### ISSUE TITLE
Improve wildcard imports

### ISSUE THREAD

**Issue by pvcraven** [pvcraven/arcade#44]
Pylint complains because files like **init**.py are doing "import *" when they should not be doing wildcard imports.

**Comment by pauleveritt** [comment]
Not sure what you'd like to do on this. It's stated quite emphatically that wildcard imports are to be avoided in nearly all cases. I think you're trying to flatten the namespace for your student developers, without putting everything in one file.

My guess, two alternatives:

- We make their lives a little harder by making them do the full import

- We make our lives a little harder by exporting each import one-by-one, manually, instead of by wildcard

**Comment by pauleveritt** [comment]
As a note, in __init__, you do:

```python
import arcade.color
import arcade.key

from arcade.application import *
```

Can we change the first two to match the last one? PyCharm gets confused and can't autocomplete arcade.color.

**Comment by pvcraven** [comment]
Is this still an issue? It seems to auto-complete for me ok right now when I use arcade.color.

**Comment by pauleveritt** [comment]
I don’t get autocomplete on arcade.color itself, though once I have arcade.color. it then completes.

—Paul

**Comment by pvcraven** [comment]
Ok, think I figured this out.

**Comment by pvcraven** [comment]
Done and closed with release 1.2.3.
